pkg/guestagent/guestagent_linux.go

```diff
@@ -244,3 +246,30 @@ func (a *agent) Info(ctx context.Context) (*api.Info, error) {
 	}
 	return &info, nil
 }
+
+const deltaLimit = time.Second * 5
```

Comment:
Not sure what the right value is, but unless the price for adjusting the time is high, I would set the threshold to 2 seconds.

And this is just personal preference, but I think `2 * time.Second` reads nicer than `time.Second * 2` because it follows the standard form of "numerical value" followed by "unit".

Reply:
I chose 5 seconds because I had never saw an evidence that skew below that might cause any problems. 

I have no benchmark data at hand, but to my best knowledge ioctl calls here are relatively cheap. Furthermore, tuning that delta won't really affect the number of update calls much. Updating according to your suggestion.